<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupeGraphique</class>
 <widget class="QWidget" name="GroupeGraphique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1211</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>10</y>
      <width>425</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="nouveauGroupe">
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="valideNouveauGroupe">
       <property name="maximumSize">
        <size>
         <width>246</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Ajouter un groupe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="listeGroupes"/>
     </item>
     <item>
      <widget class="QPushButton" name="supprimerGroupe">
       <property name="maximumSize">
        <size>
         <width>246</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableView" name="matrice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1211</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
